<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>701</width>
     <height>791</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>                                          회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>PW look</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_join">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ps_join">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>290</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="ps_look">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="join_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>640</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>join</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>460</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>이름</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_join">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>450</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="move_login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GO 로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>540</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>구분</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="div_join">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>530</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>&quot;학생&quot;,&quot;선생님&quot; 입력하세요</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>OOO님 반갑습니다</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logout_btn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qna_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Q/A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="study_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>학습하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chat_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>상담하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mypage_btn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>마이페이지</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>641</width>
       <height>571</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page_4">
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>581</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>481</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="qna_send">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>400</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>무엇이든 물어보세요</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5"/>
     <widget class="QWidget" name="page_6">
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>581</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>선생님과 상담하기</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>481</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>514</x>
         <y>430</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_7">
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>나의 학습진도</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>나의 포인트</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8"/>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>430</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>510</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ps_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>510</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="login_action">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>610</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="join_page">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>610</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
